### PR TITLE
[KTLN-571] Getting the Value of a Custom Property Defined in 'gradle.properties' in Kotlin

### PR DESCRIPTION
**Pull request by albertache1998**
Added unit tests

### CODE REVIEW

File: core-kotlin-modules/core-kotlin-advanced-3/pom.xml [hunk 1]
```diff
@@ -71,6 +71,18 @@
...
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M7</version>
+                <configuration>
+                    <environmentVariables>
+                        <CUSTOM_PROPERTY>CustomValue</CUSTOM_PROPERTY>
+                        <API_URL>https://api.example.com</API_URL>
+                    </environmentVariables>
+                </configuration>
+            </plugin>
+
```
Comment (Bwvolleyball) [inline core-kotlin-modules/core-kotlin-advanced-3/pom.xml:85]: I think this is leftover, and is probably not needed anymore?

File: kotlin-build-plugins/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+customProperty=CustomValue
+apiUrl=https://api.example.com
```
Comment (Bwvolleyball) [inline kotlin-build-plugins/gradle.properties:2]: generally, properties should be all lowercase and period separated.

```
custom.property
api.url
```

**Comment by Bwvolleyball** [comment]
retest this please

File: core-kotlin-modules/core-kotlin-advanced-3/pom.xml [hunk 3]
```diff
@@ -70,7 +70,6 @@
                     </execution>
                 </executions>
             </plugin>
-
```
Comment (KevinGilmore) [inline core-kotlin-modules/core-kotlin-advanced-3/pom.xml:73]: No reason to change this file. Let's revert it.

**Pull request closed without merging**